web/src/main/java/com/marklogic/hub/web/model/StepModel.java

```diff
@@ -23,6 +23,8 @@
 import com.marklogic.hub.util.json.JSONObject;
 import com.marklogic.hub.web.model.entity_services.JsonPojo;
 
+import java.util.Iterator;
+
 public class StepModel extends JsonPojo {
```

Comment:
do we have to extends JsonPojo?

Reply:
Nope, we absolutely don't require that thanks to your library.